Source/WebCore/loader/cache/CachedFont.cpp

```diff
@@ -77,6 +118,16 @@ void CachedFont::finishLoading(const FragmentedSharedBuffer* data, const Network
     checkNotify(metrics);
```

Comment:
I am a little surprised we don't call setLoading(false) and checkNotify() in the new error case. Presumable we want to indicate that the load has finished, still.

Reply:
`setErrorAndDeleteData()` calls `error(Status::DecodeError)` that will call both `setLoading(false)` and `checkNotify` with empty metrics.